contrib/README.md

```diff
@@ -0,0 +1,70 @@
+# Contrib scripts
+
+This directory anything which isn't exactly elfshaker but is nonetheless useful.
```

Comment:
? This directory ~~anything which~~ isn't exactly elfshaker but is nonetheless useful.

Reply:
Rewritten:
> This directory contains scripts which make use of elfshaker, for example
scripts to build a [manyclangs](https://github.com/elfshaker/manyclangs) pack.